sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py

```diff
@@ -137,6 +138,12 @@ def _from_generated(cls, generated):
         )
 
 
+class DeleteSnapshotsOption(str, Enum):
+    """The type of snapshots to delete."""
+    include = "include"
+    include_with_leased = "include-with-leased"
```

Comment:
In storage do we have a convention of upper-cased enum values?

Reply:
Nope they're usually lower-case!